script/script.js

```diff
@@ -378,6 +438,45 @@ webix.ready(function () {
             footer
         ]
     });
+
     $$("list").select("dashboard");
 
+    // binding a form to datatable
+    $$("form").bind($$("datatable"));
+
+    //simultaneously filtering by tab and header
+    $$("datatable").registerFilter(
```

Comment:
I think this piece of code can be moved inside of datatable structure, e.g.:
ready: function(){this.registerFilter(.....)}

Reply:
done